pkg/xen-tools/Dockerfile

```diff
@@ -22,14 +22,22 @@ ENV BUILD_PKGS \
     gettext iasl util-linux-dev ncurses-dev glib-dev \
     pixman-dev libaio-dev yajl-dev argp-standalone \
     linux-headers git patch texinfo tar libcap-ng-dev \
-    attr-dev flex bison cmake libusb-dev zstd-dev
+    attr-dev flex bison cmake libusb-dev zstd-dev cmake python3 g++
 ENV BUILD_PKGS_arm64 dtc-dev
 
-ENV PKGS alpine-baselayout musl-utils bash libaio libbz2 glib pixman yajl keyutils libusb xz-libs libuuid sudo zstd-libs
+ENV PKGS libgcc alpine-baselayout musl-utils bash libaio libbz2 glib pixman yajl keyutils libusb xz-libs libuuid sudo zstd-libs
 ENV PKGS_arm64 libfdt
 
 RUN eve-alpine-deploy.sh
 
+ADD https://github.com/ninja-build/ninja.git#release /ninja
+WORKDIR /ninja
+RUN python3 ./configure.py --bootstrap
```

Comment:
Do we really need all these layers? Is this build too heavy? I think you could squash all these commands into a single RUN....

Reply:
@rene QEMU now uses ninja. I couldn't find APK for it in Alpine so it was faster to build it from source. Now @deitch ponited out that it is called ninja-build. I still need to give it a try

Reply:
@rene Are you sure intermediate layers are stored? It really makes it less readable to have these long blocks of `RUN` all connected with `&&`

Reply:
Yes @christoph-zededa , that's the definition of [RUN](https://docs.docker.com/engine/reference/builder/#run) command on docker... @rucoder , I understand that you need to build ninja, that's fine, my point is that you can just group the commands (using &&) instead of use multiple RUN commands and create one layer per command....

Reply:
@rene since docker is using buildkit I have not seen them.